### DISCUSSION TITLE
Build EXE with pre set server, key, and password???

### DISCUSSION THREAD

**Discussion by jasonmacer** [rustdesk/rustdesk#4913] [General]
I do remember seeing where you can rebuild the app with your server and key included, but what about automatically setting the password to a specific password and removing the ability to set the password, or possible even see the password.

Just thinking aloud here.

**Comment by a67903830** [comment]
I also hope to have this feature, and I hope the program created by this method can hide the key.

**Comment by Cabreo** [comment]
I'm interested too
